src/components/DataReportSection/index.js

```diff
@@ -30,6 +30,8 @@ const DataReportSection = () => {
     [intl.locale]
   )
 
+  //TODO: check the bug of wrapAround
```

Comment:
add the url of the issue for future reference

Reply:
added.